app/models/xccdf.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+Dir[File.join(__dir__, 'xccdf', '*.rb')].each { |f| require f }
+
+# Represents all our models which come directly from the Xccdf XML format for
+# SCAP content
+module Xccdf
```

Comment:
My hope is to move Rule, Profile, RuleResult, etc. here as well, but it's a refactoring for another PR.